src/lang/en.json

```diff
@@ -0,0 +1,391 @@
+{
+  "blank": "",
+  "nav": {
+    "tab": {
+      "about": "About",
+      "header": "Header",
+      "mime": "MIME types",
+      "security": "Security",
+      "tags": "Tags",
+      "paths": "Paths",
+      "definitions": "Definitions"
+    },
+    "actions": {
+      "delete": "Delete data",
+      "download": {
+        "json": "JSON",
+        "yaml": "YAML"
+      }
+    }
+  },
+  "error": {
+    "title": "Uh oh!",
+    "body": "It seems like something is broken.\nSometimes these issues are caused by an invalid OpenAPI document in the cache. If this is the case, <a href=\"javascript:void(0)\" click.delegate=\"delete()\">deleting the data</a> should fix it.",
+    "data-title": "Exception data"
+  },
+  "footer-powered-by": "Powered by",
+  "form": {
+    "externalDocs": {
+      "label": "External documentation",
+      "description": {
+        "label": "Description"
+      },
+      "url": {
+        "label": "URL"
+      }
+    },
+    "paths": {
+      "label": "Paths",
+      "newItemText": "New Path",
+      "item": {
+        "label": "Path #$index: ${#:key}",
+        "operation": {
+          "methods": {
+            "get": "GET",
+            "put": "PUT",
+            "post": "POST",
+            "delete": "DELETE",
+            "options": "OPTIONS",
+            "head": "HEAD",
+            "patch": "PATCH"
+          },
+          "operationId": {
+            "label": "Operation ID"
+          },
+          "summary": {
+            "label": "Summary"
+          },
+          "description": {
+            "label": "Description"
+          },
+          "deprecated": {
+            "label": "Deprecated?",
+            "choices": {
+              "blank": ""
+            }
+          },
+          "overrides": {
+            "label": "Overrides",
+            "schemes": {
+              "label": "Protocols"
+            }
+          },
+          "tags": {
+            "label": "Tags",
+            "newItemText": "New Tag",
+            "item": {
+              "label": "Tag #$index"
```

Comment:
Perhaps this placeholder can remain here. Lets just determine a consistent way to handle these types of placeholders. I.e. what would it look like if we had the `#$index` placeholder in the template, or is this an i18next parameter?